RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+CreateEntity.swift

```diff
@@ -217,9 +233,9 @@ extension OnLedgerEntitiesClient {
 		}
 
 		let stakeAndPoolAddresses = Set(
-			stakeUnitCandidates.compactMap(\.metadata.validator?.asGeneral)
-				+ stakeClaimNFTCandidates.compactMap(\.metadata.validator?.asGeneral)
-				+ poolUnitCandidates.compactMap(\.metadata.poolUnit?.asGeneral)
+			stakeUnitCandidates.compactMap(\.metadata.validator).map { $0.embed() }
```

Comment:
(No UniFFI restrictions)

@kugel3 it is inspired by PF Case Path terminology: https://github.com/pointfreeco/swift-case-paths?tab=readme-ov-file#computed-paths

where `extract` is to try go from Enum -> Case, and inverse operation is `embed`, going from (Associated Value of) Case -> Enum.

But sure we can use `.asGeneral` again - we are "free" to use that now since "integration is complete".

Reply:
TODO: changed `func embed` -> `var asGeneral`